web-client/src/ustc-ui/Icon/WarningIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+// @ts-ignore
+import USWDSWarningIcon from '../../../../node_modules/@uswds/uswds/dist/img/usa-icons/warning.svg';
+
+export function WarningIcon() {
+  return <img alt="Warning" src={USWDSWarningIcon}></img>;
+}
```

Comment:
I tried to get this as an `svg` element rather than an `img`, like so:

```
import React from 'react';
import USWDSWarningIcon from '../../../../node_modules/@uswds/uswds/dist/img/usa-icons/warning.svg';

export function WarningIcon(props: React.SVGAttributes<SVGSVGElement>) {
  return (
    <svg {...props} dangerouslySetInnerHTML={{ __html: USWDSWarningIcon }} />
  );
}
```

It worked, but it led to weird styling. The `img` was easier to work with.